oe_theme.theme

```diff
@@ -82,7 +82,10 @@ function oe_theme_theme_suggestions_field_alter(array &$suggestions, array $vari
 
   // Add suggestions for all 'Entity reference revisions' for each target type.
   if ($variables['element']['#field_type'] === 'entity_reference_revisions') {
-    $suggestions[] = 'field__' . $variables['element']['#field_type'] . '__' . $variables['element']['#items']->getFieldDefinition()->getSetting('target_type');
+    /** @var \Drupal\field\Entity\FieldConfig $field_definition */
+    $field_definition = $variables['element']['#items']->getFieldDefinition();
+    $suggestions[] = 'field__' . $field_definition->getType() . '__' . $field_definition->getSetting('target_type');
+    $suggestions[] = 'field__' . $field_definition->getType() . '__' . $field_definition->getSetting('target_type') . '__' . $field_definition->getName();
```

Comment:
We can actually start each with:
```php
$suggestions[] = 'field__entity_reference_revisions__' . ...
```
Since we are inside that if, it will make everything more readable.